### ISSUE TITLE
codepen: gitcdn.xyz doesn't respect requested version

### ISSUE THREAD

**Issue by ngraef** [angular/material#4447]
When opening a demo in CodePen from any docs version after and including 0.10.1, the latest master version of angular material is loaded.

The following URLs are added to the CodePen resources for `0.10.1`, `0.11.0-rc1`, and `HEAD` respectively:
```
https://gitcdn.xyz/repo/angular/bower-material/v0.10.1/angular-material.js
https://gitcdn.xyz/repo/angular/bower-material/v0.11.0-rc1/angular-material.js
https://gitcdn.xyz/repo/angular/bower-material/v0.11.0-rc1-master-32ab2eb/angular-material.js
```

but they all redirect to
```
https://cdn.gitcdn.xyz/cdn/angular/bower-material/90e8fdbfb80a71bf6c0013a815a25eb056d6db49/angular-material.js
```
which is `v0.11.0-rc1-master-391479b`, the latest commit on master.

--
As far as I know, GitCDN doesn't support tag/release targeting like RawGit does. Unless a specific commit hash is provided, it serves up the latest version.

**Comment by ngraef** [comment]
@ThomasBurleson so this is the intended and desired behavior?

The one benefit I see is that CodePen examples that show issues will always use the latest version, which makes it easier to check whether something is still an issue later on. However, I find it odd that the version tag is being included in the URL if the intended behavior is just to load `.../bower_material/master/angular-material.js`.

**Comment by ThomasBurleson** [comment]
@ngraef - Thx for the followup. I will investigate correcting the URL links.

**Comment by ngraef** [comment]
Fixed by cc5333ffefb3361eed82eedb27cf92f3ebef350e

**Comment by Overdrivr** [comment]
Is it normal that several Codepen examples still have broken links ?

Both examples from [https://material.angularjs.org/latest/demo/list]
This one as well [https://material.angularjs.org/latest/demo/toast]
